docs/streams/developer-guide/dsl-api.html

```diff
@@ -3208,14 +3208,13 @@ <h5><a class="toc-backref" href="#id34">KTable-KTable Foreign-Key
 import org.apache.kafka.streams.kstream.TimeWindows;
 
 // A tumbling time window with a size of 5 minutes (and, by definition, an implicit
-// advance interval of 5 minutes). Note the explicit grace period, as the current
-// default value is 24 hours, which may be larger than needed for smaller windows.
```

Comment:
remove this note:
`Note the explicit grace period, as the current
default value is 24 hours, which may be larger than needed for smaller windows.`